examples/int-range/int_range.bal

```diff
@@ -0,0 +1,20 @@
+import ballerina/io;
+
+public function main() {
+    int[] loop1 = [];
+    // Below int:range gives a range of integers from 0 to 5 (excluding) with a distance of 2
+    // between each integer (i.e. 0, 2, 4)
+    foreach int i in int:range(0, 5, 2) {
+        loop1.push(i)
+    }
+
+    io:println(loop1);
```

Comment:
Why do we need to push to an array and print instead of printing directly?

Reply:
Was thinking since there's 2 separate loops it would be easier to differentiate if printed as 2 arrays. Shall we directly print and separate with a new line instead?